<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramDialog</class>
 <widget class="QDialog" name="HistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cmb_layer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_field">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="HistogramWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_draw">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_close">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>88</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
